addons/web/static/src/views/fields/field.js

```diff
@@ -197,19 +205,34 @@ Field.parseFieldNode = function (node, models, modelName, viewType, jsClass) {
         }
     }
 
-    if (X2M_TYPES.includes(fields[name].type)) {
+    const fields = models[modelName];
+    if (!fieldInfo.string) {
+        fieldInfo.string = fields[fieldInfo.name].string;
+    }
```

Comment:
why differently than others? For the empty string case?

Reply:
Because, it depends on fields and fieldInfo.name, that we don't have when we declare the fieldInfo.